tests/lightning_qubit/test_simulate_method.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+import math
+from typing import Sequence
+
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import PHI, THETA, LightningDevice, device_name  # tested device
+from flaky import flaky
+from pennylane.devices import DefaultExecutionConfig, DefaultQubit
+from pennylane.measurements import VarianceMP
+from scipy.sparse import csr_matrix, random_array
+
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+    from pennylane_lightning.lightning_qubit.lightning_qubit import simulate
+
+
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
+    from pennylane_lightning.lightning_kokkos.lightning_kokkos import simulate
+
+
+if device_name != "lightning.qubit" and device_name != "lightning.kokkos":
+    pytest.skip(
+        "Exclusive tests for lightning.qubit and lightning.kokkos. Skipping.",
+        allow_module_level=True,
+    )
+
+if not LightningDevice._CPP_BINARY_AVAILABLE:
+    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
+
+
+# General LightningStateVector fixture, for any number of wires.
+@pytest.fixture(
+    scope="module",
+    params=[np.complex64, np.complex128],
+)
+def lightning_sv(request):
+    def _statevector(num_wires):
+        return LightningStateVector(num_wires=num_wires, dtype=request.param)
+
+    return _statevector
```

Comment:
Is this something we would like to use in more than on pytest file? If yes, could you please move it to conftest.py?

Reply:
Yes, we use this on more than one test. I will move it

Reply:
You can do it when it is better. Just keep track of this, please.

Reply:
Ok, I think I will include it on this PR :rocket:

Reply:
Done!